### PR TITLE
feat(api): statistics of workflowruns belongs to workflow and project

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

 Add api to statistic  workflowruns

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/server/apis/v1alpha1/types.go [hunk 1]
```diff
@@ -203,3 +203,45 @@ type TLSClientConfig struct {
 	// CAData takes precedence over CAFile
 	CAData []byte `json:"caData,omitempty" bson:"caData"`
 }
+
+// StatusStats represents statistics of project or workflow.
+type StatusStats struct {
```
Comment (supereagle) [inline pkg/server/apis/v1alpha1/types.go]: Stats not only include status, but also workflow count. What about `Stats`? We can add more infos into it later.

File: pkg/server/biz/statistic/statistic.go [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+// Stats counts every StatusPhase number of wfrs, and calculate success ratio of the workflowruns.
+func Stats(list *v1alpha1.WorkflowRunList, startTime, endTime string) (*api.StatusStats, error) {
+	start, end, err := checkAndTransTimes(startTime, endTime)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Infof("stats wfrs length: %d, start time: %s, end time: %s", len(list.Items), start.String(), end.String())
+	// filter all wfr by : start < creationTimestamp end
```
Comment (supereagle) [inline pkg/server/biz/statistic/statistic.go]: // filter all wfr by : start < creationTimestamp && creationTimestamp < end

Comment (zhujian7) [reply pkg/server/biz/statistic/statistic.go]: ok

**Comment by zhujian7** [comment]
@zhujian7 PTAL

**Comment by supereagle** [comment]
> @zhujian7 PTAL

😂

**Pull request merged**